lua/pantran/curl.lua

```diff
@@ -21,6 +26,11 @@ function curl:_spawn(request, path, data, callback)
     tostring(self._url / path)
   }
 
+  if self.config.proxy.enable then
+    table.insert(args, 1, ("%s:%s"):format(self.config.proxy.host, self.config.proxy.port))
+    table.insert(args, 1, "-x")
```

Comment:
Please use the long name (`--proxy`) for readability.

Reply:
`-x` There is more flexibility to specify more proxy addresses.
example:
`curl -x http://proxy1.example.com,http://proxy2.example.com:10010 http://test.com`